receiver/splunkhecreceiver/receiver.go

```diff
@@ -202,7 +203,8 @@ func (r *splunkReceiver) Start(_ context.Context, host component.Host) error {
 	}
 
 	mx := mux.NewRouter()
-	mx.NewRoute().Path(r.config.HealthPath).HandlerFunc(r.handleHealthReq)
+	mx.NewRoute().Path(r.config.HealthPath).HandlerFunc(r.handleHealthReq).Methods("GET")
```

Comment:
I feel allowing only GETs, although appropriate, is a breaking change.  @atoulme @dmitryax is that acceptable without feature gate/deprecation process?

Reply:
I would make this a separate PR at least. @omrozowicz-splunk do you mind reverting that change?

Reply:
Sure! Done.